### DISCUSSION TITLE
3 pin fans not detected

### DISCUSSION THREAD

**Discussion by therion13** [Rem0o/FanControl.Releases#730] [Q&A] [answered]
I have 3 pin fans and i can adjust them with my motherboard so i want to make them to monitor the gpu temps but this program don't see any fans at all. only the gpu fans. Is it because alla my fans are 3 pin? they do have adjustable speed what can i do to detect them. My motherboard have 2 case fan headers. i have 3 fans on one header and 2 on the second header

**Comment by Rem0o** [comment]
See README.

**Reply by therion13** [reply]
> See README.

There is no thing on readme about my problem. i ques it does not work for 3 pin. In the program i see only gpu fan speed. all other fans are like 65535 it is like i dont have fans

**Reply by Rem0o** [reply]
The readme states that I can’t individually handle all compatibility issues which are handled by a third party open source library, which you can help contributing to.

**Reply by therion13** [reply]
> The readme states that I can’t individually handle all compatibility issues which are handled by a third party open source library, which you can help contributing to.

Ok i see. I don't think i can help you i know nothing about coding and all that. I am rooting for you i believe you will make the best fan control ever eventually. Any way i have the Arctic P12 Pressure Optimised 3-pin and Asrock Z3070m pro4. The argus monitor that recognize all my fans i don't want to use it i don't like it is say Monitoring chip Nuvoton NCT6683D i dont know maybe this is the chip you are missing in the code? good luck

**Comment by therion13** [comment, marked as answer]
Ok i just found a solution. I simple opened HWInfo and automatically detected all the fans! I didn't even press for a new check sensor again. I closed the hwinfo then and it still remember all the sensors no problem. It's working!!! Thank you very much. Just every time i open the pc i need to reapply the curve or change something for Fan control to take over the control of everything. +support from me. keep up the good work

**Reply by Bluscream** [reply]
You can save the config
